Translations/translation_EL.json

```
@@ -61,7 +61,7 @@
             "message": "Απ."
         },
         "DeviceFailedValidationWarning": {
-            "message": "Your device is most likely a counterfeit!"
+            "message": "Η συσκευή σας ίσως να μην είναι αυθεντική!!"
```

Was the double **`!!`** intended?
Or did you mean:
```suggestion
            "message": "Η συσκευή σας ίσως να μην είναι αυθεντική!"
```